index.html

```diff
@@ -630,7 +630,43 @@ <h2>DID Documents</h2>
     </section>
 
     <section>
-      <h2>DID Resolvers and DID Resolution</h2>
+      <h2>DID Resolvers</h2>
+
+      <p>
+A <a>DID resolver</a> is a software and/or hardware component that takes a
+<a>DID</a> as input and produces a <a>conforming DID document</a> as output.
+This process is called <a>DID resolution</a> and often includes resolution input
+options and resolution output metadata.
+      </p>
+      <p>
+The general process of <a>DID resolution</a> is described in
+<a href="#did-resolution"></a>. The details of implementing a
+<a>DID resolver</a> are described in the DID Resolution specification
+[[?DID-RESOLUTION]].
+      </p>
+    </section>
+
+    <section>
+      <h2>DID Dereferencers</h2>
+
+      <p>
+A <a>DID URL dereferencer</a> is a software and/or hardware component that takes
```

Comment:
Same comment about ordering above: define the process, then the component that provides that process.

Reply:
As discussed on the call, leaving this as-is for now to talk about the "nouns" in the architecture, and then what the nouns do.